### ISSUE TITLE
ENOENT: no such file or directory

### ISSUE THREAD

**Issue by ADelRosarioH** [matutter/dockerfilejs#2]
Hi @matutter!

I trying to install this module and every time I tried, I got the following error:

npm WARN whaleshot@1.0.0 No repository field.
npm ERR! Linux 4.4.0-45-generic
npm ERR! argv "/home/adelrosarioh/.nvm/versions/node/v7.1.0/bin/node" "/home/adelrosarioh/.nvm/versions/node/v7.1.0/bin/npm" "install" "dockerfilejs" "--save"
npm ERR! node v7.1.0
npm ERR! npm  v3.10.9
npm ERR! path /home/adelrosarioh/projects/whaleshot/node_modules/dockerfilejs/error.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
                                                                                                                                                                                                                   
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/adelrosarioh/projects/whaleshot/node_modules/dockerfilejs/error.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/adelrosarioh/projects/whaleshot/node_modules/dockerfilejs/error.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent
                                                                                                                                                                                                                   
npm ERR! Please include the following file with any support request:
npm ERR!     /home/adelrosarioh/projects/whaleshot/npm-debug.log

I try updating my node and npm version and still the error.

Let me know if you want me to attach the npm-debug.log

**Comment by matutter** [comment]
Thanks for the feedback, lets take a look...
So error.js is a file that used to be part of the repo but error.js isn't a file in this repo anymore. That makes me think the problem may relate to an older version of dockerfilejs laying around somehow. 

Could you try something like this to see if cleaning it up helps? (maybe run this with sudo?)
```
npm uninstall dockerfilejs
npm cache clean
npm install dockerfilejs 
```

**Comment by matutter** [comment]
I also noticed that you're running `node v7.1.0`. If you check the .travlic.yml file, I don't support that (yet). If clearing the cache doesn't work I'll test it against that version of node and add support for it.

**Comment by ADelRosarioH** [comment]
Thanks for the quick response! I'm happy to help.

![Here's a screentshot of the steps I went tru](https://cloud.githubusercontent.com/assets/3265130/20538102/1d26a49a-b0c6-11e6-90c0-094a7ff40190.PNG)

**Comment by matutter** [comment]
[7d8b71d](https://github.com/matutter/dockerfilejs/commit/7d8b71ddc4f4109370525d8dad2e302359242c3b) closes #2, looks like `npm init` did a little more work then I wanted it to.

**Comment by matutter** [comment]
[7d8b71d](https://github.com/matutter/dockerfilejs/commit/7d8b71ddc4f4109370525d8dad2e302359242c3b) closes #2, looks like `npm init` did a little more work then I wanted it to.
